docs/repos.md

```diff
@@ -4,77 +4,116 @@
+
+Both APIs implement [swagger/openapi](https://swagger.io/specification/) documentation for developers using the APIs.
+
 <div class='projects' markdown='1'>
 
-- ## [GraphKB API](https://github.com/bcgsc/pori_graphkb_api)
+- ### [GraphKB API](https://github.com/bcgsc/pori_graphkb_api)
 
     ![graphkb api](./images/graph-icon_outline.svg)
 
-    GraphKB REST API and Graph Database. The GraphKB database is implemented using [OrientDB](https://orientdb.org/).
-    It is a graph database which is used to store variants, ontologies, and the relevance
-    of this terms and variants. The KB uses strict controlled vocabulary to provide a parseable
-    and machine-readable interface for other applications to build on. The API is a REST API built
-    on node/express.
+    GraphKB REST API and Graph Database. The GraphKB database is a graph database which is used to store variants, ontologies, and the relevance of these terms and variants. The KB uses strict controlled vocabulary to provide a parseable and machine-readable interface for other applications to build on.
 
-- ## [GraphKB client](https://github.com/bcgsc/pori_graphkb_client)
+- ### [IPR API](https://github.com/bcgsc/pori_ipr_api)
 
-    ![graphkb client](./images/graphkb_graph_view_square.png)
+    ![ipr api](https://www.bcgsc.ca/gsc-logos/icon/logo-circle.png)
+
+    The Integrated Pipeline Reports (IPR) REST API manages data access to the IPR database. The API is responsible for storing and server all data for reports.
+
+</div>
 
-    The GraphKB client is the front-end web client for the GraphKB project. It is built using
-    React.js and webpack. The client is used to explore and manage content within GraphKB.
+## ETL / Data Loading
 
-- ## [GraphKB Python Adaptor](https://github.com/bcgsc/pori_graphkb_python)
+The GraphKB project also includes a loaders package which are used to import content from external knowledge bases and ontologies into GraphKB. Writing these loaders requires a strong understanding of the knowledge graph structure of GraphKB as well as the structure of the target resource. The loaders are written in javscript to be able to leverage the parser and schema JS packages used by the API and client. They are run with [NodeJS](https://nodejs.org/en/). A list of the popular supported inputs can be found in the [loading data section](./graphkb/loading_data.md).
+
+<div class='projects' markdown='1'>
+
+- ### [GraphKB Data Loaders](https://github.com/bcgsc/pori_graphkb_loader)
+
+    ![graphkb loaders](./images/graph-icon_outline.svg)
+
+    GraphKB loaders is responsible for all data import into GraphKB. Automatic Import modules are provided for a variety of external ontologies and knowledgebases such as: Ensembl, Entrez Genes, RefSeq, HGNC, Disease Ontology, NCI Thesaurus, CIViC, DoCM, etc.
```

Comment:
in general is it knowledgebase or knowledge base?

Reply:
ah, should be a space. I thought I caught all these